src/PKSim.Core/Model/AggregationFunctions.cs

```diff
@@ -28,40 +28,43 @@ public static class AggregationFunctions
 
```

Comment:
why is `BoxWhisker`**_90_**`Aggregation = boxWhiskerAggregation(`**_5_**`)`?
same for `BoxWhisker95Aggregation `(also this one is not used anywhere)